### PR TITLE
Fixed migration of deleted usernames so the don't appear as 'Guest'

### PR DESCRIPTION
**Pull request by Xeroxxx**
Unregistred/Deleted user in phpBB3 named "Bob" appears as "Guest" after migration.

This fixed the issue so "Bob" appears as "Bob".

### CODE REVIEW

File: resources/class_cache_handler.php [hunk 1]
```diff
@@ -271,7 +271,7 @@ function cache_usernames()
 	 * @param string Username used before import
 	 * @return string Username in MyBB or the old username (if provided)/'Guest' if the old UID cannot be found
 	 */
-	function username($old_uid, $old_username="")
+	function username($old_uid, $old_username)
```
Comment (JN-Jones) [inline resources/class_cache_handler.php:274]: This line shouldn't be changed as we use this function in multiple places without providing an old username.

**Comment by JN-Jones** [comment]
@Xeroxxx can you fix the issue I mentioned above?

File: resources/class_cache_handler.php (hunk 1 above)
Comment (Xeroxxx) [inline resources/class_cache_handler.php:274]: So the description above the function is wrong, isn't it?

Comment (Stefan-MyBB) [inline resources/class_cache_handler.php:274]: No, it's not wrong. It even says "if provided"...

**Comment by JN-Jones** [comment]
@Xeroxxx Any news here?

File: resources/class_cache_handler.php (hunk 1 above)
Comment (Sama34) [inline resources/class_cache_handler.php:274]: @Xeroxxx if you revert your change to that line I think the merge it good to go, thanks!

There will be issues with other modules if you don't. (:

Comment (Xeroxxx) [inline resources/class_cache_handler.php:274]: Hi,
I will have a look into it this or next week.

Greetz

**Comment by Xeroxxx** [comment]
@JN-Jones 
Hi,
I will have a look into it this or next week.

Greetz

File: resources/class_cache_handler.php (hunk 1 above)
Comment (Xeroxxx) [inline resources/class_cache_handler.php:274]: @Stefan-ST but it will be set to "" even if a username is provieded.

**Comment by JN-Jones** [comment]
I've gone ahead and pushed this directly as we don't use master and the change was only needed on feature. Thanks anyway @Xeroxxx!

**Comment by Xeroxxx** [comment]
Okay, if anyone needs this fix for phpBB to myBB he will find it ;)

**Pull request closed without merging**